EventualConsistencyDemo/Views/Movies/Movie.cshtml

```diff
@@ -84,25 +70,27 @@
                 values[field.name] = field.value;
             });
 
-            connection.invoke("SubmitOrder", parseInt(values["theatersContext_group"]) /* theater */, parseInt(values["movieId"]) /* movie */, values["times_group"] /* time */, parseInt(values["numberOfTickets"]) /* number of tickets */).catch(function (err) {
-                //connection.invoke("SubmitOrder", 1 /* theater */, 2 /* movie */, '19:00' /* time */, 5 /* number of tickets */).catch(function (err) {
+            connection.invoke("SubmitOrder"
+                , parseInt(values["theatersContext_group"]) /* theater */
+                , parseInt(values["movieId"]) /* movie */
+                , values["times_group"] /* time */
+                , parseInt(values["numberOfTickets"]) /* number of tickets */
+            ).catch(function (err) {
                 return console.error(err.toString());
             });
 
-            $('#overlay').css('display', 'block');
-
-            return false;
+            $('#overlay').show();
         });
 
-        //$('#read-reviews').click(function(e) {
-        //    e.preventDefault();
-
-        //});
+        theWindow.scroll(function(){
+            $(".down_arrow").css("opacity", 1 - $(window).scrollTop() / 250);

```

Comment:
NICE!!! :-)

Reply:
:') je meldde het in je presentatie... kleine moeite voor me om het gelijk ff te fixen ;)